frontend/src/app/main/ui/workspace/sidebar/assets/components.cljs

```diff
@@ -571,6 +594,10 @@
         (mf/use-fn
          (mf/deps file-id)
          (fn [component event]
+           ;; dnd api only allow to acces to the dataTransfer data on on-drop (https://html.spec.whatwg.org/dev/dnd.html#concept-dnd-p)
+           ;; We need to know if the dragged element is from the local library on on-drag-enter, so we need to keep the info elsewhere
+           (set-drag-data! {:local? local?})
```

Comment:
I think we don't need wrap this in an object, can we use `local?` directly as value on the drag data?

Reply:
I mean, the `drag-data*` is initialized with `false` without object wrapping so I think we should be consistent on the data stored in this atom, it is ok to have plain boolean value or object containing a single prop.

Reply:
Fixed in the initialization. I like it to be an object in case we need to keep more info there in the future.